This patch disables usage of the bundled icu library as alpine
already has 63.1
This saves about 30 MB on the final jfxwebkit.so
--- old/modules/javafx.web/src/main/native/Source/PlatformJava.cmake
+++ new/modules/javafx.web/src/main/native/Source/PlatformJava.cmake
@@ -1,4 +1,3 @@
 add_subdirectory(ThirdParty/sqlite)
-add_subdirectory(ThirdParty/icu)
 add_subdirectory(ThirdParty/libxml)
 add_subdirectory(ThirdParty/libxslt)
--- old/modules/javafx.web/src/main/native/Source/WTF/wtf/unicode/java/UnicodeJava.h
+++ new/modules/javafx.web/src/main/native/Source/WTF/wtf/unicode/java/UnicodeJava.h
@@ -18,7 +18,7 @@
 #if PLATFORM(JAVA) && OS(WINDOWS)
 typedef wchar_t UChar;
 #else
-typedef uint16_t UChar;
+//typedef uint16_t UChar;
 #endif
 
 // #ifdef UChar32
@@ -26,7 +26,7 @@
 // #endif
 
 #ifndef __UMACHINE_H__ //XXX: recheck
-typedef uint32_t UChar32;
+//typedef uint32_t UChar32;
 #endif
 
 #define U_MASK(x) ((uint32_t)1<<(x))
--- old/modules/javafx.web/src/main/native/Tools/CMakeLists.txt
+++ new/modules/javafx.web/src/main/native/Tools/CMakeLists.txt
@@ -48,6 +48,3 @@
     add_subdirectory(TestWebKitAPI)
 endif ()
 
-if (PORT STREQUAL "Java")
-    add_subdirectory(DumpRenderTree/java)
-endif ()
